crates/red_knot_python_semantic/resources/mdtest/annotations/deferred.md

```diff
@@ -0,0 +1,56 @@
+# Deferred annotations
+
+## Basic deferred annotation
+
+```pyi path=mod.pyi
+class C(object): ...
+```
+
+```py
+from mod import C
+
+reveal_type(C.__mro__)  # revealed: tuple[Literal[C], Literal[object]]
+```
```

Comment:
huh. I'm not sure what this was originally testing, but it now seems hopelessly superseded by the tests in https://github.com/astral-sh/ruff/blob/main/crates/red_knot_python_semantic/resources/mdtest/mro.md. I think we can probably just delete it?

Reply:
> huh. I'm not sure what this was originally testing

Glad it's not just me :smile:. I'll delete it.